tests/tests_pytorch/trainer/logging_/test_distributed_logging.py

```diff
@@ -194,3 +206,22 @@ def on_test_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") ->
     assert trainer.logger.logs == {"fit": 1, "validate": 1, "test": 1}
     trainer.predict(model)
     assert trainer.logger.logs == {"fit": 1, "validate": 1, "test": 1, "predict": 1}
+
+
+@pytest.mark.parametrize("distributed_env", [True, False])
+def test_logger_sync_dist(distributed_env):
```

Comment:
We have similar tests in `tests/tests_pytorch/core/test_metric_result_integration.py`. I would move this one there

Reply:
done!